src/test/java/com/wikia/webdriver/testcases/discussions/DeleteAllPostsByUserTests.java

```diff
@@ -0,0 +1,247 @@
+package com.wikia.webdriver.testcases.discussions;
+
+import com.wikia.webdriver.common.contentpatterns.MercuryWikis;
+import com.wikia.webdriver.common.contentpatterns.URLsContent;
+import com.wikia.webdriver.common.core.Assertion;
+import com.wikia.webdriver.common.core.annotations.Execute;
+import com.wikia.webdriver.common.core.annotations.InBrowser;
+import com.wikia.webdriver.common.core.drivers.Browser;
+import com.wikia.webdriver.common.core.helpers.Emulator;
+import com.wikia.webdriver.common.core.helpers.User;
+import com.wikia.webdriver.common.core.imageutilities.Shooter;
+import com.wikia.webdriver.common.core.url.UrlBuilder;
+import com.wikia.webdriver.common.remote.Discussions;
+import com.wikia.webdriver.common.remote.operations.DiscussionsOperations;
+import com.wikia.webdriver.common.templates.NewTestTemplate;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.DeleteAllButton;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.DeleteDialog;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.Post;
+import com.wikia.webdriver.elements.mercury.components.discussions.common.PostEntity;
+import com.wikia.webdriver.elements.mercury.pages.discussions.UserPostsPage;
+import org.testng.annotations.*;
+
+
+import static com.wikia.webdriver.elements.mercury.components.discussions.common.DiscussionsConstants.DESKTOP_RESOLUTION;
+
+
+/**
+ * Tests for deleting all posts by some user
+ */
+@Execute(onWikia = MercuryWikis.DISCUSSIONS_AUTO)
+public class DeleteAllPostsByUserTests extends NewTestTemplate {
+
+  private User UserWithPosts = User.USER_6;
+  private String siteId;
+
+  /**
+   * Hacky way to get rid of totally redundant loading of Special:Version.
+   * The only reason it was loaded was to extract siteId; in this case siteId is extracted from
+   * response of the same page, but parsing is via regex, once, so no need to load the page each time
+   */
+  //@Override
+  //protected void loadFirstPage() {}
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)